target/linux/ath79/files/drivers/mtd/nand/rb4xx_nand.c

```diff
@@ -301,27 +268,16 @@ static int rb4xx_nand_probe(struct platform_device *pdev)
 
 	info->chip.chip_delay	= 25;
 	info->chip.ecc.mode	= NAND_ECC_SOFT;
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,6,0)
 	info->chip.ecc.algo = NAND_ECC_HAMMING;
-#endif
 	info->chip.options = NAND_NO_SUBPAGE_WRITE;
 
 	platform_set_drvdata(pdev, info);
 
-	ret = nand_scan_ident(mtd, 1, NULL);
-	if (ret) {
-		ret = -ENXIO;
-		goto err_free_info;
-	}
-
```

Comment:
That's not a kernel version condition related change.

Reply:
Might be just a rebase issue.

Reply:
nand_scan_ident / nand_scan_tail are both deprecated, and are to be replaced by nand_scan.

Previously this code section was wrapped in "#if LINUX_VERSION_CODE < KERNEL_VERSION(4,19,0)" and instead I have changed the patch to now removed the code.

Reply:
I could edit the commit log to make this clearer?